src/components/team/Team.jsx

```diff
@@ -1,96 +1,55 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import ".//team.css";
 import ProfileCard from "../shared/team/ProfileCard";
-import Scott from "../../assets/team_pictures/scott.png";
-import Kevin from "../../assets/team_pictures/kevin.png";
-import Landseer from "../../assets/team_pictures/landseer.png";
-import Varun from "../../assets/team_pictures/varun.png";
-import Yash from "../../assets/team_pictures/yash.JPG";
-import Fahim from "../../assets/team_pictures/fahim.png";
-import Khushi from "../../assets/team_pictures/khushi.png";
-import Sarah from "../../assets/team_pictures/sarah.png";
+import { client } from "../utils/apiClient";
 import Rachit from "../../assets/team_pictures/rachit.png";
 import Rithvik from "../../assets/team_pictures/rithvik.png";
-import Grace from "../../assets/team_pictures/grace.jpeg";
-import Gunyup from "../../assets/team_pictures/gunyup.png";
+
+async function getExecs() {
+  try {
+    const response = await client.get('/executive')
+    return JSON.parse(response.data.slice(0,-4))
```

Comment:
Just wondering, what are we excluding with the `data.slice()` here?

Reply:
Besides this, everything else looks good to merge

Reply:
null, which is what I brought up cuz the api is return null at the end.

Reply:
> Besides this, everything else looks good to merge

Make sure to approve it, so I can merge. It's currently blocked <3

Reply:
> null, which is what I brought up cuz the api is return null at the end.

Ah okay, that makes sense. And yeah, my bad for forgetting to approve it :)